Misc/Test-Suite/KatharinaStyle/runTests.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+
+import unittest
+import os
+import subprocess
+import tempfile
+# import shutil
+
+
+class Test(unittest.TestCase):
+
+    def setUp(self):
+        self.fp_workdir = tempfile.mkdtemp()
+        os.system('basedir=`pwd` && cd "%s" && gapc -p "alg_dotBracket*alg_count" $basedir/../../../nodangle.gap -I $basedir/../../../' % self.fp_workdir)
+        os.system('basedir=`pwd` && cd "%s" && make -f out.mf CPPFLAGS_EXTRA="-I $basedir/../../../"' % self.fp_workdir)
+
+        self.sequences = [
+            "CCCAa+aaaUGGG",
```

Comment:
are those sequences different from the below test? If not, it might be more readable to only use the dict below and use keys here only?!